shared/js/shared-utils/parse-user-agent-string.es6.js

```diff
@@ -1,8 +1,13 @@
 module.exports = (uaString) => {
-    if (!uaString) uaString = window.navigator.userAgent
+    if (!uaString) uaString = navigator.userAgent
 
     let browser
     let version
+    let os = 'o'
+
+    if (uaString.indexOf('Windows') !== -1) os = 'w'
+    if (uaString.indexOf('Mac') !== -1) os = 'm'
+    if (uaString.indexOf('Linux') !== -1) os = 'l'
```

Comment:
In MV2 OS is saved in localStorage (only thing we are saving in localStorage 🤷‍♂️). Since localStorage is not available in service worker, instead of storing OS I just extended this method to return info about OS based on UA.